package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-square-reader-sdk",
-  "version": "1.7.2",
+  "version": "1.7.7",
```

Comment:
And this the analogous Android version number, distinct from the iOS one but tracking the (separate) SDK version numbers?  Cool.

Reply:
this should be the version of the plugin itself, unfortunately it kind of matches some of the RSDK versions but it is independent